app/adapters/telegram_adapter/outbound/photo.rb

```diff
@@ -28,7 +28,7 @@ def perform(contributor_id:, media:, message:)
             caption: optional_caption(index)
           }
         end
-        response = Telegram.bot.send_media_group(
+        response = Telegram.bots[contributor.organization.slug.underscore.to_sym || :default].send_media_group(
```

Comment:
We discussed this earlier in our call and I think there is a bit of nuance that we didn't get into.  Until now I was working under the instructions that a contributor could only be a part of one organization, but if we want to change that, let's see what issues we may run into.

We talked about always passing in the `@organization` instance variable, but we do not have access to these from our models, adapters, etc. WE schedule the messages to be sent out in an after create commit, but since a message can only ever belong to a single request and a request can only ever belong to a single organization, we can use `message.request.organization`... For the other messages we send out in response to an incoming message, we need to either look up the organization based on some param we receive in the webhook and/or the path.